WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewController.swift

```diff
@@ -171,6 +179,6 @@ final class CardReaderSettingsViewController: UIViewController {
 private extension CardReaderSettingsViewController {
 
     func configureNavigation() {
-        title = NSLocalizedString("Card Readers", comment: "Card reader settings screen title")
+        title = NSLocalizedString("Manage Card Reader", comment: "Card reader settings screen title")
```

Comment:
Same comment as above about localized strings

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3950/commits/26b6425e46cb4523f3e2f99f1b078ab941be81fc